src/components/Video/VideoPlayerWidget.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from "react-helmet"
+
+export const VideoPlayerWidget = (props) => {
+    useEffect(() => {
+        /*global cloudinary*/
+        (function () {

```

Comment:
It might be worth checking if `cloudinary` exists and logging something if it's not there. I'm thinking if folks start using this but forget to add the external cloudinary js to their page.